tuplex/core/src/HistoryServerConnector.cc

```diff
@@ -147,6 +151,35 @@ namespace tuplex {
         json obj;
         obj["job"] = job;
 
+        // add operators...
+        std::vector<json> ops;
+        //        TransformStage* trafoStage = dynamic_cast<TransformStage*>(_stage);
+        assert(plan);
+        plan->foreachStage([&](const PhysicalStage* stage) {
+            for(auto op: stage->get_ops()) {
+                json val;
+                val["name"] = op->name();
+                val["id"] = "op" + std::to_string(op->getID());
+                // @Todo: solve this...
+                val["columns"] = std::vector<std::string>();
+                val["stageid"] = stage->getID();
+                // UDF code @TODO
```

Comment:
can the TODO be removed or is there still something left todo?

Reply:
It has been removed. It was an old TODO